templates/base/.env

```diff
@@ -0,0 +1 @@
+PWA_DISABLED=false
```

Comment:
Not planning to keep any env files this time. Lets avoid having this flag itself. We can consider this as default feature.

Reply:
will remove it.